bin/spreaper

```diff
@@ -119,6 +120,14 @@ def _arguments_for_status_cluster(parser):
     parser.add_argument("cluster_name", help="the cluster name")
 
 
+def _arguments_for_list_schedules(parser):
+    """Arguments relevant for listing repair schedules. Almost the same as args for status cluster,
+    but allows cluster name to not be present
+    """
+    # It was impossible to reuse this for listing all schedules without making cluster name optional
```

Comment:
This comment can be removed now, after separating those two arguments helper methods.